ios/Podfile

```diff
@@ -1,14 +1,35 @@
-platform :ios, '11.0'
-
 require_relative '../node_modules/react-native/scripts/react_native_pods'
 require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
 
+platform :ios, '12.4'
+# install! 'cocoapods', :deterministic_uuids => false
```

Comment:
Is this line necesary?

Reply:
nope, it was supposed to be part of the upgrade but the builds would fail and the upgrade works well without it..